xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-ui/src/main/resources/XWiki/SolrSearchAdmin.xml

```diff
@@ -105,10 +105,11 @@
   #set ($discard = $xwiki.jsx.use('XWiki.SolrSearchAdmin'))
   == $services.localization.render('solr.admin.status.title') ==
 
-  (% class="xform" %)
-  ; {{html clean="false"}}&lt;label&gt;$services.localization.render('solr.admin.status.queueSize.label')&lt;/label&gt;{{/html}}##
+  (% class="xform" %)(((
+  ; {{html clean="false"}}<label>$services.localization.render('solr.admin.status.queueSize.label')</label>{{/html}}##
```

Comment:
The special XML characters were encoded for a reason. This shows that you have modified the page XML directly, which you shouldn't do. You should:

1. edit the page in your XWiki instance
1. export it as XAR
1. copy the page XML to the source tree
1. run ``mvn xar:format``
1. finally commit only the relevant changes

Reply:
Ok, I tested without special characters and it worked fine.
Now I know why they were used like this. Thanks for letting me know, I was confused about it.
Working on it now :)

Reply:
Oh yep, I just followed your steps and it turned `< >` into `&lt; &gt;` respectively. Now I am doing only the relevant changes.
Thanks as always.